packages/ui/src/2_molecules/Tooltip/Tooltip.module.css

```
@@ -0,0 +1,88 @@
+.tooltip {
+    @apply bg-gray-10 rounded p-3 text-gray-1 absolute inline-block min-w-5 max-w-72 leading-5 opacity-0;
```

```suggestion
    @apply bg-gray-10 rounded p-3 text-gray-90 absolute inline-block min-w-5 max-w-72 leading-5 opacity-0;
```
Tooltip text should be `gray-90` according to the design.

---

yep, seems it was updated later